esmvaltool/utils/cmorizers/obs/cmorize_obs_HadCRUT4.ncl

```diff
@@ -0,0 +1,202 @@
+; #############################################################################
+; ESMValTool CMORizer for HadCRUT data
+; #############################################################################
+;
+; Tier
+;    Tier 2: other freely-available dataset.
+;
+; Source
+;    https://crudata.uea.ac.uk/cru/data/temperature/
+;
+; Last access
+;    20190208
+;
+; Download and processing instructions
+;    Download the dataset "HadCRUT4" (median temperature anomalies) and
+;    the dataset "Absolute" (absolute temperatures for the base period
+;    1961-90 on a 5x5 grid).
+;
+; Caveats
+;    In contrast to the HadCRUT3 reformat script which produces temperature
+;    anomalies (relative to the 1961-90 climatology), this script calculates
+;    absolute tas by adding the climatology ("absolute.nc") to the anomalies
+;    ("HadCRUT.4.6.0.0.median.nc"). It creates two output, one with the
+;    temperature time-series and one with the climatology of the temperature
+;    anomaly.
```

Comment:
... climatology of the temperature (1961-90). (No anomaly.)

Reply:
Thank you, I changed it.